.buildkite/pipelines/e2e.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - command: .buildkite/scripts/lifecycle/pre_build.sh
+    label: Pre-Build
+    timeout_in_minutes: 10
+    agents:
+      queue: kibana-default
+    retry:
+      automatic:
+        - exit_status: '*'
+          limit: 1
+
+  - command: .buildkite/scripts/steps/build_kibana.sh
+    label: Build Kibana Distribution and Plugins
+    agents:
+      queue: n2-16-spot
+    key: build
+    if: "build.env('KIBANA_BUILD_ID') == null || build.env('KIBANA_BUILD_ID') == ''"
+    timeout_in_minutes: 60
+    retry:
+      automatic:
+        - exit_status: '-1'
+          limit: 3
+
+  - command: .buildkite/scripts/steps/functional/security_solution.sh
+    label: 'Security Solution Tests'
```

Comment:
We are missing the `APM` cypress and the `Synthetics` playwright tests here. Can we add it please?

Reply:
Pushed [52e3e5e](https://github.com/elastic/kibana/pull/152846/commits/52e3e5e45014a15c1a11c65c744b8321a65937d9)